### DISCUSSION TITLE
Allow users to provide filters for entities configuration

### DISCUSSION THREAD

**Discussion by legovaer** [dwainscheeren/dwains-lovelace-dashboard#410] [Dwains Dashboard V3 Alpha]
## Problem

Currently I have to manually add all my entities in the `entities.yaml` file in order to hide stuff that I don't want to be visible in the dashboard. Here's a (limited list) of what it looks like for me right now (FYI: my entities.yaml file has 500+ lines):

```
# Tado stuff
binary_sensor.link_badkamer_tado: &hide
  hidden: true
binary_sensor.link_gastenkamer_tado: [<<: *hide]
binary_sensor.link_eetkamer_tado:  [<<: *hide]
binary_sensor.link_inkomhal_tado: [<<: *hide]
binary_sensor.link_kantoor_tado: [<<: *hide]
binary_sensor.link_woonkamer_tado: [<<: *hide]
binary_sensor.badkamer_early_start: [<<: *hide]
binary_sensor.va2602051328_battery_state: [<<: *hide]
binary_sensor.va2602051328_connection_state: [<<: *hide]
binary_sensor.va2866948096_battery_state: [<<: *hide]
binary_sensor.va2866948096_connection_state: [<<: *hide]
binary_sensor.va3027838720_battery_state: [<<: *hide]
binary_sensor.va3027838720_connection_state: [<<: *hide]
binary_sensor.va3982632704_battery_state: [<<: *hide]
binary_sensor.va3982632704_connection_state: [<<: *hide]
binary_sensor.voordeur_audio_detected: [<<: *hide]
binary_sensor.voordeur_crossline_detected: [<<: *hide]
```

## Solution / Motivation

It would be nice if there is a possibility to be able to apply filters like in [the HA recorder configuration](https://www.home-assistant.io/integrations/recorder/#configure-filter):

```
hidden:
    domains:
      - input_boolean
      - sensor
    entity_globs:
      - input_boolean.tado_*
    entities:
      - sensor.something_cool
      - sensor.something_else
```

Using the same mechanism, it should be possible to do exactly the same with `favorite` and `disable` ...

**Comment by dwainscheeren** [comment]
That would me something for a later 3.* version. But for now you can fast hide/disable them in the ui? 
![image](https://user-images.githubusercontent.com/3868853/155704950-313df395-3651-4501-827a-43a1ea2cad8b.png)

**Reply by legovaer** [reply]
When in mobile, if I "hide" an entity, the entire dashboard reloads. This means I have to click on "edit" again..

**Reply by dwainscheeren** [reply]
Yeah that is fixed in alpha 6, don't forget where are still in alpha fase ;) :)

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
